2023-TK/1959 Altena.txt

```diff
@@ -0,0 +1,34 @@
+https://www.gemeentealtena.nl/fileadmin/Altena_bestanden/PDF_bijlagen/Verkiezingen/22_november_2023_Tweede_Kamer_Verkiezingen/Processen_verbaal_per_stembureau/1959_PV_Stembureau_1_Gereformeerde_Kerk__d_Ontmoeting__1__TK23.pdf

```

Comment:
Dit moet zijn: https://www.gemeentealtena.nl/fileadmin/Altena_bestanden/PDF_bijlagen/Verkiezingen/22_november_2023_Tweede_Kamer_Verkiezingen/Uitslagen_per_stembureau/1959_Uitslag_Stembureau_1_Gereformeerde_Kerk__d_Ontmoeting__1__TK23.pdf

Reply:
> Dit moet zijn: https://www.gemeentealtena.nl/fileadmin/Altena_bestanden/PDF_bijlagen/Verkiezingen/22_november_2023_Tweede_Kamer_Verkiezingen/Uitslagen_per_stembureau/1959_Uitslag_Stembureau_1_Gereformeerde_Kerk__d_Ontmoeting__1__TK23.pdf

Die link lijkt mij niet goed. Op de webpagina staan zowel Uitslag-PDF's als Proces-verbaal-PDF's.

Mijn browser downloadt https://www.gemeentealtena.nl/fileadmin/Altena_bestanden/PDF_bijlagen/Verkiezingen/22_november_2023_Tweede_Kamer_Verkiezingen/Processen_verbaal_per_stembureau/1959_PV_Stembureau_1_Gereformeerde_Kerk__d_Ontmoeting__1__TK23.pdf zonder probleem... wat werkt er niet?

Reply:
Ik ga 'm even opnieuw proberen. Ik denk dat mijn script struikelde over de (windows?) regeleindes.

Reply:
Excuses voor het ongemak, ik zit inderdaad op Windows. Ik maakte de patches evenwel direct via de browser, die editor neemt kennelijk dan ook mijn regeleindes over. Ik zal eens kijken of ik dat kan instellen/aanpassen.

Reply:
Het lag inderdaad aan de regel-eindes. Zie d2526e59a308c0b2cb60cb098580edb8274635b6

Is dat in te stellen in de Github editor? Ik kan ook gewoon het script wat relaxter maken...

Reply:
#10